src/test/ui/suggestions/option-content-move.fixed

```diff
@@ -8,7 +8,7 @@ impl LipogramCorpora {
     pub fn validate_all(&mut self) -> Result<(), char> {
         for selection in &self.selections {
             if selection.1.is_some() {
-                if selection.1.as_ref().unwrap().contains(selection.0) {
+                if selection.1.as_ref().as_mut().unwrap().contains(selection.0) {
```

Comment:
I'm not sure how to stop it from doing this

Reply:
What happens if you use `span_suggestion_verbose` twice instead of one `span_suggestions`?